newsletter-bundle/contao/modules/ModuleUnsubscribe.php

```diff
@@ -85,10 +85,19 @@ protected function compile()
 				'name' => 'unsubscribe_' . $this->id,
 				'label' => $GLOBALS['TL_LANG']['MSC']['securityQuestion'],
 				'inputType' => 'captcha',
-				'eval' => array('mandatory'=>true)
+				'eval' => array('mandatory'=>true, 'required'=>true)
 			);
 
-			$objWidget = new FormCaptcha(FormCaptcha::getAttributesFromDca($arrField, $arrField['name']));
+			/** @var class-string<FormCaptcha> $strClass */
+			$strClass = $GLOBALS['TL_FFL']['captcha'] ?? null;
+
+			// Fallback to default if the class is not defined
+			if (!class_exists($strClass))
+			{
+				$strClass = 'FormCaptcha';
```

Comment:
```suggestion
				$strClass = FormCaptcha::class;
```

Reply:
`ModuleRegistration` will need the same treatment for consistency.

Reply:
Changed in b7a79e3.